### PR TITLE
OLM 2.0.1

### PR DESCRIPTION
**Pull request by dvaldivia**
Updating the OLM definition to `2.0.0`

### CODE REVIEW

File: olm-catalog/minio-operator/2.0.0/minio-operator.v2.0.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+    - Cloud Storage
+  maintainers:
+    - name: MinIO Developers
+      email: dev@min.io
+  provider:
+    name: 'MinIO, Inc.'
+  labels: {}
+  selector:
+    matchLabels: {}
+  links:
```
Comment (harshavardhana) [inline olm-catalog/minio-operator/2.0.0/minio-operator.v2.0.0.clusterserviceversion.yaml:95]: We can add links for 

```
MinIO Subnet?
```

File: olm-catalog/minio-operator/2.0.0/minio-operator.v2.0.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+            "requestAutoCert": false
+          }
+        }
+      ]
+spec:
+  displayName: MinIO Operator
+  description: >
+    ## About the managed application
+
+    MinIO object storage platform enables building high performance data
```
Comment (harshavardhana) [inline olm-catalog/minio-operator/2.0.0/minio-operator.v2.0.0.clusterserviceversion.yaml:48]: The content should be from our website

```
MinIO's high-performance, software-defined object storage suite
enables customers to build cloud-native data infrastructure for
machine learning, analytics, and application data workloads.
```

File: olm-catalog/minio-operator/2.0.0/minio-operator.v2.0.0.clusterserviceversion.yaml (hunk 1 above)
Comment (dvaldivia) [reply olm-catalog/minio-operator/2.0.0/minio-operator.v2.0.0.clusterserviceversion.yaml:95]: I think we only have the `pricing` page https://min.io/pricing is this one ok?

Comment (harshavardhana) [reply olm-catalog/minio-operator/2.0.0/minio-operator.v2.0.0.clusterserviceversion.yaml:95]: Yeah works @dvaldivia

Comment (dvaldivia) [reply olm-catalog/minio-operator/2.0.0/minio-operator.v2.0.0.clusterserviceversion.yaml:96]: done

**Pull request merged**